### ISSUE TITLE
Outdated Release Status (2020/12/18)

### ISSUE THREAD

**Issue by Cernelius** [triplea-game/triplea#8452]
In the 
https://triplea-game.org/map/over-the-top/
maps page, I can see:
>(Requires pre-release 1.9.0.0.6322 or higher)

I believe it is a long time since 1.9.0.0.6322 hasn't been a pre-release if it ever was a release.

I believe this is a general problem.

**Comment by CharlieAtlas** [comment]
Is that something on the page?  Or is it something on the maps themselves?  I am guessing it is on the website, who has access to it?

I would guess just removing the 'pre-' would be the solution?

**Comment by Cernelius** [comment]
My suggestion is removing it entirely. Such things are virtually doomed to be forgotten, thus become obsolete. For any clarification, I guess it would best be given by whover added it.

**Comment by Cernelius** [comment]
It is also confusing because, whereas I'm pretty sure that is just for using the button (meaning that only since TripleA 1.9.0.0.6322 it exists the feature of having the program responding to the command you give by clicking on the "Download with TripleA" button), that is not clear, and some may easily wonder that is referring to a TripleA minimum version required by the featured map.

If keeping it, and as long as I'm correct, I would reword it to something like:
_Direct download of the map from this page is supported only since TripleA 1.9.0.0.6322._

Otherwise, a, more concise, exortation to download the latest TripleA before hitting the button.

**Comment by DanVanAtta** [comment]
My 2 cents, I'd just remove it.  Very few players should be on such an old version. If so, they are going to see other problems that are far more severe (like frozen battle calcs for any sub combat in a bot game). 


cc: @jdimeo

**Comment by jdimeo** [comment]
AFAIK I didn't explicitly add this to Over The Top. I just have:
```
<triplea minimumVersion="1.9"/>
```
in the game file and it looks like that was "picked up" when the page was auto-generated. I think this was the current version back when I started work on the map.

**Comment by CharlieAtlas** [comment]
So who has access to the website?  That seems to be website code to me, not map specific.  It was the same for several maps that i checked.

**Comment by Cernelius** [comment]
> AFAIK I didn't explicitly add this to Over The Top. I just have:
> 
> ```
> <triplea minimumVersion="1.9"/>
> ```
> 
> in the game file and it looks like that was "picked up" when the page was auto-generated. I think this was the current version back when I started work on the map.

I don't think anything was picked up. I'm almost sure all at this issue would be the same if you set the game's version at 2.0 or whatever.

**Comment by DanVanAtta** [comment]
Ah, I was under the quick impression that the description text there was carried over from the specific map. It appears that instead it is the website that is adding that text. Therefore it seems we simply need to remove it from the website: github.com/triplea-game/triplea-game.github.io/

**Comment by CharlieAtlas** [comment]
Do we want it gone?  or just to say version 1.9 or higher?

**Comment by Cernelius** [comment]
> Ah, I was under the quick impression that the description text there was carried over from the specific map. It appears that instead it is the website that is adding that text. Therefore it seems we simply need to remove it from the website: https://github.com/triplea-game/triplea-game.github.io/

This issue should be opened in that queque then.
